docs/2_getting_started_session.md

```diff
@@ -143,31 +143,16 @@ dds data ls --project "<Project ID>" --tree
 
  - [ ] TASK: Use the ls command to verify that all the data has being uploaded succesfully.
 
-#### Release project and download data
+#### Download data
 
 > Currently DDS does not support to resume downloads, if your download is interrupted you will need to restart it.
 
-1. In order for researchers to get access to the data, you need to [change the status](https://scilifelabdatacentre.github.io/dds_cli/project/#dds-project-access) to `Available`. If a researcher tries to download data and the project is still in progress they will get an error.
 
-~~~
-dds project status release --project "<Project ID>"
-~~~
-
-![Message shown after project release](https://scilifelabdatacentre.github.io/dds_cli/_images/dds-project-status-release.svg)
-##
-
-After releasing the project, any current researcher associated with the project will receive a notification by email.
-
-##
-3. To [download](https://scilifelabdatacentre.github.io/dds_cli/data/#dds-data-get) the full project contents:
+To [download](https://scilifelabdatacentre.github.io/dds_cli/data/#dds-data-get) the full project contents:
 ~~~
 dds data get --get-all --project "<Project ID>" 
 ~~~
 
-
```

Comment:
Were we going to add an 'Optional' header to the section about downloading?

Reply:
mmmm but that downloading is not mentioned in 3). And I feel like knowing the basic command to get data is usefull

Reply:
Fair point! I'm not pushed either way. Thanks for all the PRs this afternoon